PreLoad.cmake

```diff
@@ -20,9 +20,9 @@ if (NOT DEFINED ENV{CLION_IDE} AND NOT DEFINED ENV{XCODE_IDE})
 endif()
 
 # Check if environment is polluted.
-if (NOT $ENV{CFLAGS} STREQUAL ""
-    OR NOT $ENV{CXXFLAGS} STREQUAL ""
-    OR NOT $ENV{LDFLAGS} STREQUAL ""
+if (NOT "$ENV{CFLAGS}" STREQUAL ""
```

Comment:
Quotes are not necessary but I am fine with them.

Reply:
Here - yes, but they are necessary in case there is no `NOT`:

```cmake
if ($ENV{foo} STREQUAL "foo")
    message(WARNING "foo")
endif()
if (NOT $ENV{bar} STREQUAL "bar")
    message(WARNING "bar")
endif()
```

```sh
$ cmake .
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at CMakeLists.txt:1 (if):
  if given arguments:

    "STREQUAL" "foo"

  Unknown arguments specified


-- Configuring incomplete, errors occurred!
See also "/tmp/test-cmake/CMakeFiles/CMakeOutput.log".
```

```
$ foo=foo cmake .
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning at CMakeLists.txt:2 (message):
  foo


-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/test-cmake
```

So I prefer to make the style the same, i.e.`NOT "$ENV{VAR}" STREQUAL ""` and `"$ENV{VAR}" STREQUAL ""`

Reply:
Okay, that makes sense then. Thanks for clarifying.